### PR TITLE
Mood classifier

### PR DESCRIPTION
**Pull request by JeremybellEU**
Mood classifier for songs #46

### CODE REVIEW

File: src/main/java/me/moodcat/mood/Mood.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import me.moodcat.database.embeddables.VAVector;
+
+/**
+ * A mood represents a vector in the valence-arousal plane which will be attached to song.
+ *
+ * @author JeremybellEU
+ */
+public enum Mood {
+
+    ANGRY(0.0, 0.0),
```
Comment (JWGmeligMeyling) [inline src/main/java/me/moodcat/mood/Mood.java]: Can you put some estimated values here? Given that we range in [-1, 1] and take the points from the graph in the book?

File: src/main/java/me/moodcat/mood/classifier/MoodClassifier.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+public class MoodClassifier implements Predictor<Song, VAVector> {
+
+    @Override
+    public VAVector predict(final Song song) {
+        final AcousticBrainzData data = song.getFeatures();
+
+        final double valence = data.getLowlevel().getDissonance().getMean()
+                + data.getTonal().getKeyStrength();
+
+        final double arousal = data.getRhythm().getBpm()
```
Comment (JWGmeligMeyling) [inline src/main/java/me/moodcat/mood/classifier/MoodClassifier.java]: this does nothing right?

File: pom.xml [hunk 3]
```diff
@@ -28,6 +28,7 @@
 		<pmd.version>3.2</pmd.version>
 		<findbugs.version>3.0.1</findbugs.version>
 		<querydsl.version>3.6.0</querydsl.version>
+		<recommendation-library.version>2.0.0</recommendation-library.version>
```
Comment (JWGmeligMeyling) [inline pom.xml]: Is the correct version deployed now?

**Comment by JWGmeligMeyling** [comment]
LGTM :+1:

**Comment by Azkle** [comment]
Can be merged.

**Comment by JeremybellEU** [comment]
Yes, this is ready to be merged, as @Azkle already is working on tests. LGTM

**Pull request merged**